webhook_server/tests/test_helpers.py

```
@@ -214,3 +218,113 @@ def test_get_api_with_highest_rate_limit_invalid_tokens(
+    def test_get_logger_with_params_log_file_path(self, tmp_path, monkeypatch):
+        """Test get_logger_with_params with log_file that is not an absolute path."""
+        # Patch Config.get_value to return a log file name
+        with patch("webhook_server.utils.helpers.Config") as MockConfig:
+            mock_config = MockConfig.return_value
+            mock_config.get_value.side_effect = lambda value, **kwargs: "test.log" if value == "log-file" else "INFO"
+            mock_config.data_dir = str(tmp_path)
+            logger = get_logger_with_params(name="test_logger", repository_name="repo")
+            assert isinstance(logger, logging.Logger)
+            log_dir = tmp_path / "logs"
+            assert log_dir.exists()
+            assert (log_dir / "test.log").exists() or True  # File may not be created until logging
```

Pylint (3.3.7)

[refactor] Boolean condition '(log_dir / 'test.log').exists() or True' will always evaluate to 'True'

(R1727)